submissions/Nemooochka/frogger-game/app.js

```diff
@@ -0,0 +1,112 @@
+const
+    CELL_WIDTH = 101;
+CELL_HEIGHT = 83;
+PIC_OFFSET = 19;
+PIC_WIDTH = 80;
+FIELD_WIDTH = CELL_WIDTH * 5;
+START_POS_X = CELL_WIDTH * 2;
+START_POS_Y = CELL_HEIGHT * 5 - PIC_OFFSET;
+MIN_SPEED = 80;
+MAX_SPEED = 200;
+
+
+const Enemy = function(y, player) {
+    this.x = 0;
+    this.y = y;
+    this.speed = this.randomSpeed();
+    this.sprite = 'images/enemy-bug.png';
+    this.player = player;
+};
+
+Enemy.prototype.update = function(dt) {
+    this.x += this.speed * dt;
+    if(this.x > FIELD_WIDTH) {
+        this.x = 0;
+        this.speed = this.randomSpeed();
+    }
+    this.checkCollision();
+};
+
+Enemy.prototype.randomSpeed = function() {
```

Comment:
Usually function/method names contain verbs. This is helpful when you read code and need to understand what happens quickly. How about `getRandomSpeed`?